pkg/config/expand.go

```diff
@@ -86,11 +86,7 @@ func (dc *DeploymentConfig) expandBackends() {
 				}
 			}
 			if be.Type == "gcs" && !be.Configuration.Has("prefix") {
-				prefix := blueprint.BlueprintName
-				if deployment, err := blueprint.DeploymentName(); err == nil {
-					prefix += "/" + deployment
-				}
-				prefix += "/" + string(grp.Name)
+				prefix := fmt.Sprintf("%s/%s/%s", bp.BlueprintName, bp.DeploymentName(), grp.Name)
```

Comment:
Is this always going to be executed after we validate blueprint ? Previous behavior is we check if there is error or not while getting deployment name.